src/Serializer/AbstractItemNormalizer.php

```diff
@@ -503,6 +512,7 @@ protected function getAttributeValue($object, $attribute, $format = null, array
         $context['api_attribute'] = $attribute;
         $propertyMetadata = $this->propertyMetadataFactory->create($context['resource_class'], $attribute, $this->getFactoryOptions($context));
 
+        // BC to be removed in 3.0
```

Comment:
what should be removed?

Reply:
The whole try-catch except for the getValue.